standard-inventory-qcow2

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python
+
+import argparse
+import errno
+import json
+import os
+import shutil
+import shlex
+import signal
+import socket
+import subprocess
+import sys
+import tempfile
+import time
+import traceback
+import distutils.util
+
```

Comment:
It's worth including a comment here that this is a copy from standard-test-roles with patches, and eventually shouldn't be included in this repo.

Reply:
Done.